### PR TITLE
(#169) - Some couchdb-harness fixes

### PR DESCRIPTION
**Pull request by marten-de-vries**
The title says it all

### CODE REVIEW

**Comment by marten-de-vries** [comment]
Tests for this are in https://github.com/nick-thompson/couchdb-harness/pull/7

**Comment by marten-de-vries** [comment]
With pouchdb/couchdb-harness#7 merged, couchdb-harness passes in Travis for this PR. :)

File: lib/routes/db.js [hunk 1]
```diff
@@ -34,6 +34,15 @@ module.exports = function (app) {
...
   app.delete('/:db', function (req, res, next) {
+    if (req.query.rev) {
+      return utils.sendJSON(res, 400, {
+        error: 'bad_request',
+        reason: (
+          "You tried to DELETE a database with a ?rev= parameter. " +
+          "Did you mean to DELETE a document instead?"
+        )
+      });
+    }
```
Comment (nolanlawson) [inline lib/routes/db.js]: @janl has [a blog post](http://blog.couchdb.org/2014/04/08/the-little-things1-do-not-delete/) about this feature. Neat to see it implemented here. :)

**Comment by nolanlawson** [comment]
Holy crap, that's amazing. express-pouchdb has come such a long way in such a short amount of time. Nice work!

**Comment by nolanlawson** [comment]
0bdc5e07bfb4d00bf896c6ca253a929594324661

**Pull request closed without merging**